static/index.html

```diff
@@ -1,67 +1,217 @@
-                        <li class="list-group-item">Slice your file as normal</li>
-                        <li class="list-group-item">Click "G&#x3E;" button to start direct upload to the printer</li>
-                        <li class="list-group-item">Click "Upload and print" - the "Upload"-only is not supported</li>
-                        <li class="list-group-item">Enjoy printing with ankerctl 😉</li>
-                    </ol>
-                    <br><span class="fw-light">Hint: <kbd>Ctrl+Shift+G</kbd> will slice and open the upload to printer window</span>
-                </div>
-                <div class="col-12 col-lg-12 text-center">
-                    <img src="/static/img/setup/prusaslicer-1.png" class="img-thumbnail" alt="step 1">
-                    <img src="/static/img/setup/prusaslicer-2.png" class="img-thumbnail mt-2" alt="step 2">
-                    <img src="/static/img/setup/prusaslicer-3.png" class="img-thumbnail mt-2" alt="step 3">
-                    <img src="/static/img/setup/prusaslicer-4.png" class="img-thumbnail mt-2" alt="step 4">
-                </div>
+        <footer class="container my-3 text-center bg-body-tertiary rounded-3 text-center text-lg-start">
+            <div class="p-1 text-center">
+                &copy;
+                <script>
+                    document.write(new Date().getFullYear());
+                </script>
+                Copyright:
+                <a
+                    class="text-light"
+                    href="https://github.com/Ankermgmt/ankermake-m5-protocol"
+                    alt="ankerctl"
+                    rel="noopener noreferrer nofollow"
+                    target="_blank"
+                    >ankerctl</a
+                ><br /><a
+                    href="https://github.com/Ankermgmt/ankermake-m5-protocol"
+                    class="btn btn-link btn-sm"
+                    alt="View on GitHub"
+                    rel="noopener noreferrer nofollow"
+                    target="_blank"
+                    ><i class="bi-github"></i> View on GitHub</a
+                >
             </div>
-        </div>
-        <script src="/static/vendor/bootstrap-5.3.0-alpha3-dist/js/bootstrap.bundle.min.js"></script>
-        <script src="/static/vendor/cash.min.js"></script>
-        <script type="text/javascript" src="/static/vendor/jmuxer.min.js"></script>
-        <script type="text/javascript" src="/static/ankersrv.js"></script>
+            <script type="text/javascript" src="{{ url_for('static', filename='vendor/bootstrap-5.3.0-alpha3-dist/js/bootstrap.bundle.min.js')}}"></script>
+            <script type="text/javascript" src="{{ url_for('static', filename='vendor/cash.min.js') }}"></script>
+            <script type="text/javascript" src="{{ url_for('static', filename='vendor/jmuxer.min.js') }}"></script>
+            <script type="text/javascript" src="{{ url_for('static', filename='ankersrv.js') }}"></script>
+        </footer>
```

Comment:
actually, in html5, `<footer>` is the last element loaded in the dom model. so putting scripts in the footer is actually one of the best places because just putting them in front of the closing `</body>` doesn't ensure that they are loaded last :)

Reply:
✅